packages/itwinui-react/src/core/utils/components/Resizer.tsx

```diff
@@ -216,99 +234,80 @@ export const Resizer = (props: ResizerProps) => {
       <div
         data-iui-resizer='top-left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'nw-resize',
-        }}
+        style={{ cursor: 'nw-resize' }}
       />
       <div
         data-iui-resizer='top'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 'n-resize',
-        }}
+        style={{ cursor: 'n-resize' }}
       />
       <div
         data-iui-resizer='top-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'ne-resize',
-        }}
+        style={{ cursor: 'ne-resize' }}
       />
       <div
         data-iui-resizer='right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          right: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'e-resize',
-        }}
+        style={{ cursor: 'e-resize' }}
       />
       <div
         data-iui-resizer='bottom-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'se-resize',
-        }}
+        style={{ cursor: 'se-resize' }}
       />
       <div
         data-iui-resizer='bottom'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 's-resize',
-        }}
+        style={{ cursor: 's-resize' }}
       />
       <div
         data-iui-resizer='bottom-left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'sw-resize',
-        }}
+        style={{ cursor: 'sw-resize' }}
       />
       <div
         data-iui-resizer='left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          left: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'w-resize',
-        }}
+        style={{ cursor: 'w-resize' }}
```

Comment:
nit: Is there any advantage with defining this also in `resizerStyles`?

Reply:
I used plain CSS to get access to selectors. Inline `style` is ok here, because the `cursor` is different for every resizer.